Library/Context.ts

```diff
@@ -54,21 +54,7 @@ class Context {
     }
 
     private _loadInternalContext() {
-        // note: this should return the sdk package.json
-        let packageJsonPath = path.resolve(__dirname, "../../package.json");
-
-        if (!Context.sdkVersion) {
-            Context.sdkVersion = "unknown";
-            try {
-                let packageJson = JSON.parse(fs.readFileSync(packageJsonPath, "utf8"));
-                if (packageJson && typeof packageJson.version === "string") {
-                    Context.sdkVersion = packageJson.version;
-                }
-            } catch (exception) {
-                Logging.info("unable to read app version: ", exception);
-            }
-        }
-
+        Context.sdkVersion = APPLICATION_INSIGHTS_SDK_VERSION;
```

Comment:
Just curious - why do you change the version to hardcoded rather than extract from package json?

Reply:
Looking at Statsbeat data we can tell package json logic fails a lot, this will ensure we send correct version all the times